packages/common/driver-definitions/tsconfig.json

```diff
@@ -1,11 +1,11 @@
 {
 	"extends": [
 		"../../../common/build/build-common/tsconfig.base.json",
-		"../../../common/build/build-common/tsconfig.cjs.json",
+		"../../../common/build/build-common/tsconfig.bundler.json",
```

Comment:
This is needed because the auto-gwenerated type test files have bare imports (no extensions). Once that is fixed this should be changed.